services_app/src/main/java/org/opendatakit/services/sync/actions/activities/AbsSyncBaseActivity.java

```diff
@@ -428,8 +429,13 @@ private void onFirstLaunch() {
         PropertiesSingleton props = getProps();
         props.setProperties(Collections.singletonMap(CommonToolProperties.KEY_FIRST_LAUNCH, Boolean.toString(false)));
 
-        androidx.appcompat.app.AlertDialog alertDialog = new androidx.appcompat.app.AlertDialog
-                .Builder(this)
+
+        /**
+         * MaterialAlertDialogBuilder is standard for all ODK-X Apps
+         * OdkAlertDialogStyle present in AndroidLibrary is used to style this dialog
+         */
+
+        androidx.appcompat.app.AlertDialog alertDialog = new MaterialAlertDialogBuilder(getApplicationContext(),R.style.OdkXAlertDialogStyle)
```

Comment:
Why isn't the type simply MaterialAlertDialogBuilder instead of androidx.appcompat.app.AlertDialog?

Reply:
UPDATED: why isn't the type "AlertDialog"? Why the fully qualified " androidx.appcompat.app.AlertDialog"